### PR TITLE
Synced internal changes for 1.7.3

### PR DESCRIPTION
**Pull request by sid-github**

### CODE REVIEW

File: EarlGrey/Common/GREYAnalytics.h [hunk 1]
```diff
@@ -56,17 +56,16 @@
  *  Sends an Analytics Event hit based on protocol specified in:
  *  @see https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide#event
  *
- *  @param trackingID  The tracking ID for the created Analytics Event payload.
- *  @param clientID    The client ID for the created Analytics Event payload.
- *  @param category    The category value to be used for the created Analytics Event payload.
- *  @param subCategory The sub-category value to be used for the created Analytics Event payload.
- *  @param valueOrNil  The value to be used for the created Analytics Event payload. The value
- *                     can be @c nil to indicate that value is not to be added to the payload.
+ *  @param trackingID  The tracking ID under which to track this event.
```
Comment (tirodkar) [inline EarlGrey/Common/GREYAnalytics.h]: There's 2 spaces here between ID and The.

File: EarlGrey/Common/GREYAnalyticsDelegate.h [hunk 2]
```diff
@@ -27,14 +27,14 @@
  *
  *  @param trackingID  The tracking ID under which to track this event.
```
Comment (tirodkar) [inline EarlGrey/Common/GREYAnalyticsDelegate.h]: 2 spaces again.

File: Tests/FunctionalTests/Sources/FTRBaseAnalyticsTest.m [hunk 3]
```diff
@@ -107,16 +107,16 @@ + (void)setExpectedAnalyticsRequestsCount:(NSInteger)count {
...
-                     subCategory:(NSString *)subCategory
-                           value:(NSNumber *)valueOrNil {
+                          action:(NSString *)action
+                           value:(NSString *)value {
   NSAssert([NSThread isMainThread], @"The tests expects that Analytics delegate is invoked on "
                                     @"main thread.");
   gTotalHitsReceived += 1;
   [gOriginalAnalyticsDelegate trackEventWithTrackingID:trackingID
                                                 clientID:clientID
                                               category:category
```
Comment (tirodkar) [inline Tests/FunctionalTests/Sources/FTRBaseAnalyticsTest.m]: Is this aligned right?

File: EarlGrey/Common/GREYAnalytics.h (hunk 1 above)
Comment (sid-github) [inline EarlGrey/Common/GREYAnalytics.h]: Done

File: EarlGrey/Common/GREYAnalyticsDelegate.h (hunk 2 above)
Comment (sid-github) [inline EarlGrey/Common/GREYAnalyticsDelegate.h]: Done

File: Tests/FunctionalTests/Sources/FTRBaseAnalyticsTest.m (hunk 3 above)
Comment (sid-github) [inline Tests/FunctionalTests/Sources/FTRBaseAnalyticsTest.m]: Done

**Pull request merged**